src/data/providers/balancer-api/modules/pool-state/index.ts

```diff
@@ -39,6 +43,29 @@ export class Pools {
       }
     }`;
 
+    readonly boostedPoolStateWithUnderlying = `
+    query GetPool($id: String!, $chain: GqlChain!) {
+      poolGetPool(id:$id, chain:$chain) {
+        id
+        address
+        type
+        protocolVersion
+        poolTokens {
+          index
+          address
+          decimals
+          balance
+          underlyingToken {
+            address
+            decimals
+          }
+        }
+        dynamicData {
```

Comment:
I don't think we need this field?

Reply:
You are right. I think I can actually remove the changes in the api, since I am not using the api in my tests, as we decided to use mock data. I'll remove it.